tests/Domain/Service/MessagePresenterTest.php

```diff
@@ -26,21 +26,21 @@ public function test_convert_newline_to_paragraph_converts_single_line()
     public function test_convert_newline_to_paragraph_converts_regular_multi_lines()
     {
         $content = "My simple content\nWith some other content. ";
-        $expectedContent = "<p>My simple content</p>\n<p>With some other content. </p>";
+        $expectedContent = '<p>My simple content<br />With some other content. </p>';
         $this->assertEqualHtml($expectedContent, $this->messagePresenter->present($content));
     }
 
     public function test_convert_newline_to_paragraph_works_with_crlf_multi_lines()
     {
         $content = "My simple content\r\nWith some other content. ";
-        $expectedContent = "<p>My simple content</p>\n<p>With some other content. </p>";
+        $expectedContent = '<p>My simple content<br />With some other content. </p>';
         $this->assertEqualHtml($expectedContent, $this->messagePresenter->present($content));
     }
 
     public function test_convert_newline_to_paragraph_works_with_cr_multi_lines()
     {
         $content = "My simple content\rWith some other content. ";
-        $expectedContent = "<p>My simple content</p>\n<p>With some other content. </p>";
+        $expectedContent = '<p>My simple content<br />With some other content. </p>';
         $this->assertEqualHtml($expectedContent, $this->messagePresenter->present($content));
     }
```

Comment:
Sorry to bother but we are still missing the new test case which is supposed to test that 2 (or more?) consecutive line feeds should create a paragraph.

Reply:
No, you’re right actually. I’do that.

Reply:
Done